Dockerfile

```
@@ -27,7 +27,8 @@ ARG SRC_DIR
 ARG PKG_DIR
 
 RUN apt update
```

We can merge the multiple `RUN` blocks like the followings.
```suggestion
RUN apt update; \
            apt install -yq musl-dev ca-certificates; \
            update-ca-certificates; \
            mkdir -p $PKG_DIR/conf $PKG_DIR/bin 
```